src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,161 @@
+metaDescription: 'A transition guide about a change in how New Relic handles OpenTelemetry cumulative sum metrics, and how to update your queries and alerts for that change.'
+redirects: 
+---
+
+Our OTLP metric pipeline is being upgraded to better support how we handle [cumulative metrics](/docs/data-apis/understand-data/metric-data/cumulative-metrics). As part of this upgrade we're transitioning the underlying [metric data type](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types) for [monotonic cumulative sums](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#sums) from a `gauge` type to the `cumulativeCount` type. The `cumulativeCount` type allows you to query both the cumulative and delta value associated with reported data. 
+
+Starting March 14, 2023, new accounts will have access to this new `cumulativeCount` type, but if your account was sending cumulative monotonic sums prior to March 14th, you won't have access to that metric type until June 13. On that date, the use of our `gauge` metric type for cumulative sums will be deprecated and reach end-of-life (EoL), and your cumulative sums will be stored as the `cumulativeCount` type. 
+
+**This 90 day EoL period is to give you time to update queries and alerts that make use of the affected metrics.** The sections below will teach you: 
+
+* How to know if your account is impacted
+* How to prepare for the EoL
+* How to preview the `cumulativeCount` functionality 
+
+<Callout variant="important">
+Cumulative sums that are non-monotonic will **not** be affected.
+</Callout>
+
+## Check your account for impact [#impact]
+
+If your New Relic account was sending OTLP cumulative sum metrics to New Relic before March 14, 2023, then you'll be impacted by the EoL on June 13, 2023. 
+
+To check if your account is impacted, run the following NRQL query:
+
+```
+FROM NrIntegrationError SELECT count(*) 
+WHERE newRelicFeature = 'cumulativeSumAsGaugeEoL' 
+SINCE 24 hours ago
+```
+
+The query will return results if you've reported metrics impacted by this EoL in the last 24 hours. 
+
+To investigate further, you can view the event message to see which metrics are affected:
+
+```
+FROM NrIntegrationError SELECT uniques(metricName)
+WHERE newRelicFeature = 'cumulativeSumAsGaugeEoL' 
+LIMIT MAX SINCE 24 hours ago
+```
+
+Any monitoring or alerting involving these metrics will need to be reviewed and updated prior to the EoL date. The next section contains suggested actions for impacted metrics. 
+
+## What to do if you're impacted [#what-to-do]
+
+On June 13, the reporting of OTLP monotonic cumulative sum metrics will switch from using the `gauge` to `cumulativeCount`. If you're using those metrics, you'll want to prepare for that. For example, you may want to create duplicate alert conditions with the new metric type so that, when the switch happens, you won't have a disruption (and we explain how to do that below).
```

Comment:
```suggestion
On June 13, the reporting of OTLP monotonic cumulative sum metrics will switch from using the `gauge` to `cumulativeCount` metric type. If you're using those metrics, you'll want to prepare for that. For example, you may want to create duplicate alert conditions with the new metric type so that, when the switch happens, you won't have a disruption (and we explain how to do that below).
```